src/routes/open/container/Open.jsx

```diff
@@ -1,27 +1,33 @@
 // @flow
 import queryString from 'query-string'
-import * as React from 'react'
+import React, { useEffect, useState } from 'react'
 import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
 
+import Opening from '../../opening'
 import Layout from '../components/Layout'
 
-import actions, { type Actions, type AddSafe } from './actions'
+import actions, { type Actions /* , type AddSafe */ } from './actions'
```

Comment:
there are some commented blocks in this file. Just that.

Reply:
done!